src/analyses/interval_domain.cpp

```diff
@@ -64,6 +64,7 @@ void interval_domaint::transform(
   ai_baset &,
   const namespacet &ns)
 {
+  // clang-format off
```

Comment:
This seems excessive use of `clang-format off` (several occurrences). May I ask what undesirable formatting clang-format produces in this case?

Reply:
clang-format is absolutely doing the right thing here, but I wasn't sure I should be re-formatting several hundred lines of code (some of these inconsistently formatted switch/case statements span a lot of code). I would be easy enough for me to ditch this commit (those `clang-format off/on` are all in a separate commit) and apply proper formatting instead.

Reply:
I have a preference for reformatting it properly right away. Turning off the formatter over hundreds of lines will not make it better in future.

Reply:
Done.